composer.json

```
@@ -20,7 +20,9 @@
     "require": {
         "php": "^7.2",
         "cocur/slugify": "^1.4 || ^2.0 || ^3.0 || ^4.0",
-        "sonata-project/doctrine-extensions": "^1.1",
+        "sonata-project/exporter": "^1.11.0 || ^2.0",
```

I know the result is the same, just for the sake of consistency with other constraints:
```suggestion
        "sonata-project/exporter": "^1.11 || ^2.0",
```